### PR TITLE
update Implosion Compressor to GuiBase + fix duplication

### PR DESCRIPTION
**Pull request by alexpic57**
also added the hologram

### CODE REVIEW

File: src/main/java/techreborn/tiles/multiblock/TileImplosionCompressor.java [hunk 1]
```diff
@@ -40,10 +42,12 @@
 import techreborn.client.container.builder.BuiltContainer;
 import techreborn.client.container.builder.ContainerBuilder;
 import techreborn.init.ModBlocks;
+import techreborn.tiles.TileRollingMachine;
```
Comment (drcrazy) [inline src/main/java/techreborn/tiles/multiblock/TileImplosionCompressor.java]: Why do you need this?

Comment (alexpic57) [reply src/main/java/techreborn/tiles/multiblock/TileImplosionCompressor.java]: forgot to remove it i think, i was trying stuff and i think i c/c in the wrong place

Comment (drcrazy) [reply src/main/java/techreborn/tiles/multiblock/TileImplosionCompressor.java]: You can make another commit to your branch with fix for unnecessary import. Your pull request will update automatically, as far as I know.

**Comment by modmuss50** [comment]
You have used spaces and not tabs. I think it will be easier for me to reformat the whole project after the merge.

**Comment by modmuss50** [comment]
Nice, looks good. Thanks 👍

**Pull request merged**